about-peer-review/pyopensci-related-joss-ropensci.md

```diff
@@ -0,0 +1,47 @@
+# How Are JOSS, rOpenSci and pyOpenSci Related? 
+
+## How is pyOpenSci different from JOSS and other review processes?
+
+pyOpenSci aligns closely with the broad mission of JOSS to provide maintainers with credit for their efforts in developing the open source
+tools necessary to support reproducible open science workflows. However, 
+we are not a typical publisher or Journal. Rather we are a community 
+that provides support for both a diverse group of software maintainers and 
+long term maintenance of our packages.
+
+### Why Two Review Processes JOSS and pyOpenSci? 
```

Comment:
```suggestion
## How is pyOpenSci different from JOSS and other review processes?
```
As in that structure I suggested, I'd get rid of two headlines at the top of the page, one right after the other, and instead replace this "Why two review processes" which reads very conversationally with the second headline that says the same thing while better emphasizing the need for different reviews